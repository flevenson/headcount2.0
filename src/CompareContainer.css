.compare-container {
  display: grid;
  grid-template-columns: repeat(auto-fit, minmax(230px, 1fr));
  grid-auto-flow: row;
  background: #FFFACDD9;
  margin: 0 auto;
}

.card-div {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  max-width: 220px;
  margin: 8px auto;
  height: 270px;
  box-shadow: 0 0 5px 1px #0000006e;
  border-radius: 5px;
  background-color: #9AC5C5
}