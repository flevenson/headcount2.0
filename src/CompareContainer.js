import React from 'react';

const CompareContainer = ( { display } ) => {



  return (
    <div>

    </div>
    )
}

export default CompareContainer