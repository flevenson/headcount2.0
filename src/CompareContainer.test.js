import React from 'react';
import { shallow } from 'enzyme';

import CompareContainer from './CompareContainer';

describe('CompareContainer', () => {


  it('should match a snapshot', () => {
    const display = [{
      location: 'Turing',
      stats: {
        '2018' : 1
      }
    },
    {
      location: 'Turing',
      stats: {
        '2018' : 1
      }
    }
    ];
    const wrapper = shallow(<CompareContainer display={ display }/>);
    expect(wrapper).toMatchSnapshot();


  });



  
});