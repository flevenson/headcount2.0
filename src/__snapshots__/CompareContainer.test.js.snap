// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompareContainer should match a snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CompareContainer
    display={
        Array [
            Object {
              "location": "Turing",
              "stats": Object {
                "2018": 1,
              },
            },
            Object {
              "location": "Turing",
              "stats": Object {
                "2018": 1,
              },
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        Array [
          <Card
            className="card[object Object]"
            handleCompare={undefined}
            location="Turing"
            stats={
                        Object {
                                    "2018": 1,
                                  }
            }
/>,
          <Card
            className="card[object Object]"
            handleCompare={undefined}
            location="Turing"
            stats={
                        Object {
                                    "2018": 1,
                                  }
            }
/>,
        ],
        <CompareCard
          avgCard={undefined}
/>,
      ],
      "className": "compare-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "0",
        "nodeType": "function",
        "props": Object {
          "className": "card[object Object]",
          "handleCompare": undefined,
          "location": "Turing",
          "stats": Object {
            "2018": 1,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "1",
        "nodeType": "function",
        "props": Object {
          "className": "card[object Object]",
          "handleCompare": undefined,
          "location": "Turing",
          "stats": Object {
            "2018": 1,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "avgCard": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          Array [
            <Card
              className="card[object Object]"
              handleCompare={undefined}
              location="Turing"
              stats={
                            Object {
                                          "2018": 1,
                                        }
              }
/>,
            <Card
              className="card[object Object]"
              handleCompare={undefined}
              location="Turing"
              stats={
                            Object {
                                          "2018": 1,
                                        }
              }
/>,
          ],
          <CompareCard
            avgCard={undefined}
/>,
        ],
        "className": "compare-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "0",
          "nodeType": "function",
          "props": Object {
            "className": "card[object Object]",
            "handleCompare": undefined,
            "location": "Turing",
            "stats": Object {
              "2018": 1,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "1",
          "nodeType": "function",
          "props": Object {
            "className": "card[object Object]",
            "handleCompare": undefined,
            "location": "Turing",
            "stats": Object {
              "2018": 1,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "avgCard": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
